deprecated.md

```diff
@@ -0,0 +1,9 @@
+## Stuff need attention
```

Comment:
can this become a github issue instead of a new file?

Reply:
No problem. Deleted..